compiler/test-resources/repl/importFromObj

```diff
@@ -14,8 +14,8 @@ val res0: scala.collection.mutable.ListBuffer[Int] = ListBuffer(1, 2, 3)
 scala> import util.foo
 1 | import util.foo
   |             ^^^
-  |             value foo is not a member of util - did you mean util.Left?
+  |             value foo is not a member of util - did you mean util.Try?
```

Comment:
Could you explain this one to me? `import util.Left` is invalid? Or did it change because `Try` has a higher priority on the list of suggestions?

Reply:
> Or did it change because Try has a higher priority on the list of suggestions?

Something like that yeah. Ideally it should just not offer any suggestion at all when there's nothing close to what the user has written.